pytest.ini

```diff
@@ -1,4 +1,5 @@
 [pytest]
+python_files = spectrum/*.py
```

Comment:
possible conflict of intent with `--assert=plain`.

my intent is to turn unhelpful errors like:

```
    def test_function():
>       assert f() == 4
E       AssertionError  f() == 4
```

into Pytest's:

```
    def test_function():
>       assert f() == 4
E       assert 3 == 4
E        +  where 3 = f()
```

unfortunately Pytest needs to be told that assertions outside of `test_*` files should have their assertions introspected as well.